momepy/preprocessing.py

```diff
@@ -4,12 +4,19 @@
 import math
 import operator
 import warnings
+import os
+
+PYGEOS = True
+if os.environ.get("USE_PYGEOS", "1") == "0":
```

Comment:
You can't expect True if the variable is not set.

Reply:
I had forgotten I had this set as a global `env var` in my `.profile`. It is academic anyway and will close this now.